src/Helpers/File_System_Helper.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace JustCoded\WP\Composer\Helpers;
+
+/**
+ * Class File_System_Helper
+ *
+ * @package JustCoded\WP\Composer\Helpers
+ */
+class File_System_Helper {
+
+	/**
+	 * @param: source folder $src
+	 * @param: destination folder $dst
+	 */
+	public static function copy_dir( $src, $dst ) {
+		$dir = opendir( $src );
+		@mkdir( $dst );
+		while ( false !== ( $file = readdir( $dir ) ) ) {
+			if ( ( '.' !== $file ) && ( '..' !== $file ) ) {
+				if ( is_dir( $src . '/' . $file ) ) {
+					self::copy_dir( $src . '/' . $file, $dst . '/' . $file );
+				} else {
+					copy( $src . '/' . $file, $dst . '/' . $file );
+				}
+			}
```

Comment:
we refactored this yesterday. please remove so deep hierarchy

Reply:
returned back